posts/signals.py

```
@@ -101,3 +104,19 @@ def decrement_post_vote_counts(sender, instance: PostVotes, **kwargs):
         Post.objects.filter(id=post.id).update(upvotes=F("upvotes") - 1)
     elif instance.value == PostVotes.DOWNVOTE:
         Post.objects.filter(id=post.id).update(downvotes=F("downvotes") - 1)
+
+@receiver(post_save, sender=Post)
+def trigger_post_create_notifications(sender, instance: Post, created: bool, **kwargs):
+    """Initiate notification when a post is newly created."""
+    if not created:
+        return
+    try:
+        send_push_notification_to_post_creator.delay(instance.id)
+    except Exception as exc:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)